error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`

Seems to me that d4eea7211c9c3a5f4845990648df86d783d4f786 broke the build:
```
error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`
 --> .cargo/registry/src/github.com-1ecc6299db9ec823/async-tungstenite-0.4.1/src/compat.rs:6:18
  |
6 | use futures_io::{AsyncRead, AsyncWrite};
  |                  ^^^^^^^^^  ^^^^^^^^^^ no `AsyncWrite` in the root
  |                  |
  |                  no `AsyncRead` in the root

error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`
 --> .cargo/registry/src/github.com-1ecc6299db9ec823/async-tungstenite-0.4.1/src/handshake.rs:3:18
  |
3 | use futures_io::{AsyncRead, AsyncWrite};
  |                  ^^^^^^^^^  ^^^^^^^^^^ no `AsyncWrite` in the root
  |                  |
  |                  no `AsyncRead` in the root

error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`
 --> .cargo/registry/src/github.com-1ecc6299db9ec823/async-tungstenite-0.4.1/src/stream.rs:9:18
  |
9 | use futures_io::{AsyncRead, AsyncWrite};
  |                  ^^^^^^^^^  ^^^^^^^^^^ no `AsyncWrite` in the root
  |                  |
  |                  no `AsyncRead` in the root

error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/async-tungstenite-0.4.1/src/lib.rs:51:18
   |
51 | use futures_io::{AsyncRead, AsyncWrite};
   |                  ^^^^^^^^^  ^^^^^^^^^^ no `AsyncWrite` in the root
   |                  |
   |                  no `AsyncRead` in the root

error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/async-tungstenite-0.4.1/src/tokio.rs:11:18
   |
11 | use futures_io::{AsyncRead, AsyncWrite};
   |                  ^^^^^^^^^  ^^^^^^^^^^ no `AsyncWrite` in the root
   |                  |
   |                  no `AsyncRead` in the root

error[E0432]: unresolved imports `futures_io::AsyncRead`, `futures_io::AsyncWrite`
  --> .cargo/registry/src/github.com-1ecc6299db9ec823/async-tungstenite-0.4.1/src/tokio.rs:23:22
   |
23 |     use futures_io::{AsyncRead, AsyncWrite};
   |                      ^^^^^^^^^  ^^^^^^^^^^ no `AsyncWrite` in the root
   |                      |
   |                      no `AsyncRead` in the root

error: aborting due to 6 previous errors
```

Pretty much anything of relevance in `futures-io` is behind the `std` feature from what I can tell, so you may want to enable it. If I enable it the compilation goes through smoothly.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,7 +27,7 @@ features = ["async-std-runtime", "tokio-runtime", "gio-runtime", "async-tls", "a
 [dependencies]
 log = "0.4"
 futures-util = { version = "0.3", default-features = false, features = ["async-await", "sink", "std"] }
-futures-io = { version = "0.3", default-features = false }
+futures-io = { version = "0.3", default-features = false, features = ["std"] }
 pin-project = "0.4"
 
 [dependencies.tungstenite]

